Smtpapi/HeaderTests/TestUtils.cs

```diff
@@ -57,5 +58,17 @@ public void TestEncodeNonAsciiCharacters()
 
             Assert.AreEqual(expected, result);
         }
+
+        [Test]
+        public void TestLicenseFileYear()
+        {
+            string test = "Copyright (c) 2014-2016 SendGrid, Inc.";

```

Comment:
Can you please modify this to pull this string from the License file in the repository?